openage/convert/processor/export/media_exporter.py

```diff
@@ -122,6 +123,12 @@ def _export_blend(export_request, sourcedir, exportdir, blend_mode_count=None):
                 exportdir[export_request.targetdir],
                 f"{export_request.target_filename}{idx}.png"
             )
+        source_format = export_request[export_request.source_filename.rindex("."):]
+        source_size = os.path.getsize(source_file)
+        target_size = os.path.getsize(export_request.targetdir+export_request.target_filename)
+        logging.basicConfig(level=logging.DEBUG)
```

Comment:
You don't need this line if you import the openage logger. It handles the setup for you.